.travis.yml

```diff
@@ -39,42 +40,60 @@ matrix:
     env: INSTALL_DEPENDS="pip==10.0.1 setuptools==30.3.0"
 
 before_install:
+  - python -m pip install --upgrade pip virtualenv
```

Comment:
The preinstalled `pytest` clashed with `pytest-xdist` but didn't get upgraded by `pytest-xdist`'s dependencies for some reason. Virtualenv starts clean.